services/cd-service/pkg/service/overview.go

```diff
@@ -494,7 +494,8 @@ func (o *OverviewServiceServer) subscribe() (<-chan struct{}, notify.Unsubscribe
 func (o *OverviewServiceServer) update(s *repository.State) {
 	r, err := o.getOverviewDB(context.Background(), s)
 	if err != nil {
-		panic(err)
+		logger.FromContext(context.Background()).Error("error getting overview:", zap.Error(err))
```

Comment:
last time I tried this, nothing was logged.
Just to be sure: did you try that the logging works here?